tests/.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'expect': true
```

Comment:
Good notes all around; I don't think I actually use the `expect` global anywhere so removing it seems like the right way to go.